terraform/modules/aws-brig-email-sending/README.md

```diff
@@ -0,0 +1,33 @@
+Terraform module: Brig to enable email sending
+==============================================
+
+State: __experimental__
+
+Wire-server's "brig" components needs to send emails. This can either be done by configuring an
+SMTP server (Option 1), or by using AWS resources (Option 2).
+
+This terraform module enables brig to send emails using option 2. In addition, it configures *MAIL FROM* for outgoing
+emails, but does not enable incoming emails (possible solution: `aws_ses_receipt_rule`).
+
+AWS resources: SES, SQS, SNS, DNS
+
+
+#### Important note
+
+This module causes Terraform to store sensitive data in the `.tfstate` file. Hence, encrypting the state should be
+mandatory.
+
+
+#### How to use the module
+
+```hcl
+module "brig_email_sending" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws-brig-email-sending?ref=develop"
+  
+  environment = "staging"
```

Comment:
```suggestion
  environment = "dev"
```

Really nit-picky 🙈

Reply:
it defaults to `dev` actually, that is why I decided to give it a different value, otherwise it could be omitted altogether. But then, the example doesnt really makes sense anymore.